client/src/main/java/org/apache/falcon/client/FalconClient.java

```diff
@@ -150,6 +150,7 @@ public FalconClient(String falconUrl, Properties properties) throws FalconCLIExc
             client.resource(UriBuilder.fromUri(baseUrl).build());
             authenticationToken = getToken(baseUrl);
         } catch (Exception e) {
+        	e.printStackTrace();
```

Comment:
It was changed like this throw new FalconCLIException("Unable to initialize Falcon Client object. Cause : " + e.getMessage(), e); 

@baishuo  Can you please close this pull request if it is no longer required.